blaze/server/server.py

```diff
@@ -406,3 +406,42 @@ def compserver():
         'data': result,
         'names': expr.fields
     })
+
+@api.route('/add', methods=['POST', 'HEAD', 'OPTIONS'])
+@crossdomain(origin='*', methods=['POST', 'HEAD', 'OPTIONS'])
+@authorization
+def addserver():
+    content_type = request.headers['content-type']
+    matched = mimetype_regex.match(content_type)
+
+    if matched is None:
+        return 'Unsupported serialization format %s' % content_type, 415
+
+    try:
+        serial = _get_format(matched.groups()[0])
+    except KeyError:
+        return (
+            "Unsupported serialization format '%s'" % matched.groups()[0],
+            415,
+        )
+
+    try:
+        payload = serial.loads(request.data)
```

Comment:
since we are just passing k:v pairs, do you think it would make sense to encode this as query params, is there a type of resource that can have a comma in it?